packages/x-date-pickers/src/StaticNextDatePicker/StaticNextDatePicker.tsx

```diff
@@ -0,0 +1,247 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { StaticNextDatePickerProps } from './StaticNextDatePicker.types';
+import { useNextDatePickerDefaultizedProps } from '../NextDatePicker/shared';
+import { datePickerValueManager } from '../DatePicker/shared';
+import { useStaticPicker } from '../internals/hooks/useStaticPicker';
+import { CalendarPickerView } from '../internals/models';
+import { renderDateView } from '../internals/utils/viewRenderers';
+
+type StaticDatePickerComponent = (<TDate>(
+  props: StaticNextDatePickerProps<TDate> & React.RefAttributes<HTMLDivElement>,
+) => JSX.Element) & { propTypes?: any };
+
+const VIEW_LOOKUP = {
+  day: renderDateView,
+  month: renderDateView,
+  year: renderDateView,
+};
+
+const StaticNextDatePicker = React.forwardRef(function StaticNextDatePicker<TDate>(
```

Comment:
It does seem that we are no longer using the forwarded ref anywhere. 🤔 

Below is the diff resolving the following error:
```
Warning: forwardRef render functions accept exactly two parameters: props and ref. Did you forget to use the ref parameter?
```

```diff
diff --git a/packages/x-date-pickers/src/StaticNextDatePicker/StaticNextDatePicker.tsx b/packages/x-date-pickers/src/StaticNextDatePicker/StaticNextDatePicker.tsx
index a504802f4..f274989a8 100644
--- a/packages/x-date-pickers/src/StaticNextDatePicker/StaticNextDatePicker.tsx
+++ b/packages/x-date-pickers/src/StaticNextDatePicker/StaticNextDatePicker.tsx
@@ -1,4 +1,3 @@
-import * as React from 'react';
 import PropTypes from 'prop-types';
 import { StaticNextDatePickerProps } from './StaticNextDatePicker.types';
 import { useNextDatePickerDefaultizedProps } from '../NextDatePicker/shared';
@@ -7,17 +6,13 @@ import { useStaticPicker } from '../internals/hooks/useStaticPicker';
 import { CalendarPickerView } from '../internals/models';
 import { renderDateView } from '../internals/utils/viewRenderers';
 
-type StaticDatePickerComponent = (<TDate>(
-  props: StaticNextDatePickerProps<TDate> & React.RefAttributes<HTMLDivElement>,
-) => JSX.Element) & { propTypes?: any };
-
 const VIEW_LOOKUP = {
   day: renderDateView,
   month: renderDateView,
   year: renderDateView,
 };
 
-const StaticNextDatePicker = React.forwardRef(function StaticNextDatePicker<TDate>(
+const StaticNextDatePicker = function StaticNextDatePicker<TDate>(
   inProps: StaticNextDatePickerProps<TDate>,
 ) {
   const defaultizedProps = useNextDatePickerDefaultizedProps<
@@ -41,7 +36,7 @@ const StaticNextDatePicker = React.forwardRef(function StaticNextDatePicker<TDat
   });
 
   return renderPicker();
-}) as StaticDatePickerComponent;
+};
 
 StaticNextDatePicker.propTypes = {
   // ----------------------------- Warning --------------------------------

```

Reply:
I forgot to pass it to the root indeed :grimacing: , I'll do it